common/management/commands/project_external_updates.py

```diff
@@ -24,25 +24,28 @@ def handle(self, *args, **options):
 
 def get_project_github_links():
     from civictechprojects.models import ProjectLink
-    return ProjectLink.objects.filter(link_name='link_coderepo', link_url__icontains='github.com/')\
+    repo_links = ProjectLink.objects.filter(link_name='link_coderepo', link_url__icontains='github.com/')\
         .exclude(link_project__isnull=True)
+    branch_links = ProjectLink.objects.filter(link_url__icontains='github.com/').filter(link_url__icontains='/tree/')\
+        .exclude(link_name='link_coderepo', link_project__isnull=True)
```

Comment:
Duh. You're right - this didn't need to change.

Reply:
It will only pick up links with the name 'link_coderepo' and that filters out any additional branch urls that get added as links. I will remove that condition from the query altogether.